tests/contrib/aiobotocore/test.py

```diff
@@ -0,0 +1,235 @@
+        finally:
+            s3.close()
+
+    @MotoService('sqs')
+    @asyncio.coroutine
+    def test_sqs_client(self):
+        sqs = self.session.create_client('sqs', region_name='us-east-1', endpoint_url=MOTO_ENDPOINT_URL)
+        try:
+            tracer = get_dummy_tracer()
+            writer = tracer.writer
+            Pin(service=self.TEST_SERVICE, tracer=tracer).onto(sqs)
+
+            yield from sqs.list_queues()
+
+            spans = writer.pop()
+            assert spans
+            span = spans[0]
+            eq_(len(spans), 1)
+            eq_(span.get_tag('aws.region'), 'us-east-1')
+            eq_(span.get_tag('aws.operation'), 'ListQueues')
+            eq_(span.get_tag(http.STATUS_CODE), '200')
+            eq_(span.service, "test-aiobotocore-tracing.sqs")
+            eq_(span.resource, "sqs.listqueues")
+        finally:
+            sqs.close()
+
+    @MotoService('kinesis')
+    @asyncio.coroutine
+    def test_kinesis_client(self):
+        kinesis = self.session.create_client('kinesis', region_name='us-east-1', endpoint_url=MOTO_ENDPOINT_URL)
+
+        try:
+            tracer = get_dummy_tracer()
+            writer = tracer.writer
+            Pin(service=self.TEST_SERVICE, tracer=tracer).onto(kinesis)
+
+            yield from kinesis.list_streams()
+
+            spans = writer.pop()
+            assert spans
+            span = spans[0]
+            eq_(len(spans), 1)
+            eq_(span.get_tag('aws.region'), 'us-east-1')
+            eq_(span.get_tag('aws.operation'), 'ListStreams')
+            eq_(span.get_tag(http.STATUS_CODE), '200')
+            eq_(span.service, "test-aiobotocore-tracing.kinesis")
+            eq_(span.resource, "kinesis.liststreams")
+        finally:
+            kinesis.close()
+
+    @MotoService('kinesis')
+    @asyncio.coroutine
+    def test_unpatch(self):
+        kinesis = self.session.create_client('kinesis', region_name='us-east-1', endpoint_url=MOTO_ENDPOINT_URL)
+        try:
+            tracer = get_dummy_tracer()
+            writer = tracer.writer
+            Pin(service=self.TEST_SERVICE, tracer=tracer).onto(kinesis)
+
+            unpatch()
+
+            yield from kinesis.list_streams()
+            spans = writer.pop()
+            assert not spans, spans
+        finally:
+            kinesis.close()
+
+    @MotoService('sqs')
+    @asyncio.coroutine
+    def test_double_patch(self):
+        sqs = self.session.create_client('sqs', region_name='us-east-1', endpoint_url=MOTO_ENDPOINT_URL)
+
+        try:
+            tracer = get_dummy_tracer()
+            writer = tracer.writer
+            Pin(service=self.TEST_SERVICE, tracer=tracer).onto(sqs)
+
+            patch()
+            patch()
+
+            yield from sqs.list_queues()
+
+            spans = writer.pop()
+            assert spans
+            eq_(len(spans), 1)
+        finally:
+            sqs.close()
+
+    # TODO: could not get lambda working with moto server as it uses a url with
```

Comment:
Tried the mock `@MotoService('lambda')` with botocore no patch, doesn't work either. Since this is an issue due to the mocking library we might want to skip this test. Anyway for the moment aiobotocore supports officially only s3, great work on it by the way 😄